setup.php

```diff
@@ -58,6 +58,36 @@ function testPing($address)
 
     function makeInstallation($s)
     {
+
+		if ($s['oko_ip_ok'] == "true") {
```

Comment:
l'indentation me parait énorme, c'est normal ?

Reply:
Of course not